v2/coord/network.go

```diff
@@ -234,16 +234,13 @@ func (h *NodeHandler) PutValue(ctx context.Context, r Value, q int) error {
 	panic("not implemented")
 }
 
-func CloserNodesResponse(k KadKey, nodes []peer.AddrInfo) kad.Response[KadKey, ma.Multiaddr] {
-	infos := make([]kad.NodeInfo[KadKey, ma.Multiaddr], len(nodes))
+func CloserNodeIDs(nodes []peer.AddrInfo) []kadt.PeerID {
```

Comment:
That's redundant with `SliceOfAddrInfoToSliceOfKadPeerID` in `conversion.go`

Reply:
Yep, I did spot that and forgot to fix it